tests/test_vnet_bitmap.py

```diff
@@ -0,0 +1,249 @@
+from swsscommon import swsscommon
+import time
+import test_vnet as vnet
+
+
+# Define fake platform for "DVS" fixture, so it will set "platform" environment variable for "orchagent".
+# It is needed in order to enable platform specific "orchagent" code for testing "bitmap" VNET implementation.
+DVS_FAKE_PLATFORM = "mellanox"
+
+
+'''
+Implements "check" APIs for the "bitmap" VNET feature.
+These APIs provides functionality to verify whether specified config is correcly applied to ASIC_DB.
+Such object should be passed to the test class, so it can use valid APIs to check whether config is applied.
+'''
+class VnetBitmapVxlanTunnel(object):
```

Comment:
Can we have this in the same `test_vnet.py` file, considering bitmap and vrf as two different implementation for same feature?.

Reply:
We can't, because "vrf" and "bitmap" implementations are under "if platform == ..." condition in orchagent.
So, to enable that code we need to run "vs" container with "platform" environment variable set to different values.
"DVS" fixture, which creates container has scope set to "module", as a result container is started per each pytest module, that is why we need two different modules (files).

Reply:
The class definition? Its fine if you prefer to keep it in this file, but I was thinking more like if we add anything in future, we would need to modify two files with implementation

Reply:
So, what you suggest is to move ```VnetBitmapVxlanTunnel``` class implementation to common ```test_vnet.py``` file, so we have implementation in one file.
And as for the ```TestVnetBitmapOrch``` class, we can still keep it in this file as it is now.
Is my understanding correct?

Reply:
yes, that is my suggestion

Reply:
It makes sense, I already pushed required changes.